presto-main/src/main/java/io/prestosql/cost/CostCalculatorWithEstimatedExchanges.java

```diff
@@ -68,9 +68,18 @@ public PlanCostEstimate calculateCost(PlanNode node, StatsProvider stats, CostPr
 
     private static PlanCostEstimate addExchangeCost(PlanCostEstimate costEstimate, LocalCostEstimate estimatedExchangeCost)
     {
+        // Exchange memory estimates are imprecise, because they don't take into account whether current node is streaming, accumulating or a join.
```

Comment:
I don't understand it. Exchange is always streaming. Please improve comment

Reply:
Yes, exchange is streaming. But it still can allocate memory (eg buffers)

Reply:
what is a `current node` in this sentence?

Reply:
"current node" is the `node` for which `io.prestosql.cost.CostCalculatorWithEstimatedExchanges#calculateCost` was invoked.
Note that `CostCalculatorWithEstimatedExchanges` and `CostCalculatorWithEstimatedExchanges.ExchangeCostEstimator` are _not_ recursive.